packages/twenty-server/src/engine/workspace-manager/workspace-sync-metadata/workspace-sync-metadata.service.ts

```diff
@@ -168,4 +182,15 @@ export class WorkspaceSyncMetadataService {
       storage,
     };
   }
+
+  private async enableFeatureFlag(
+    featureFlagKey: FeatureFlagKey,
+    workspaceId: string,
+  ) {
+    await this.featureFlagRepository.save({
+      key: featureFlagKey,
```

Comment:
I'm curious about this one, if we don't have a unicity constraint on the key (which I think we don't), won't it insert multiple times the same key (with false and true potentially), this could lead to issues during the fetch (if we get the first one for example)